src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -26,25 +26,115 @@ public ScorerPipelineStep(Var<IDataView> data, Var<ITransformModel> model)
         public Var<ITransformModel> Model { get; }
     }
 
+
+    /// <summary>
+    /// LearningPipeline class is used to define the steps needed to perform desired machine learning task.<para/>
```

Comment:
>LearningPipeline [](start = 8, length = 16)

Class names should appear in `<see` tags`. However in this case, given that this is describing this class, I might just replace "LearningPipeline class is used" with "This class is used..."